### ISSUE TITLE
#4 is broken

### ISSUE THREAD

**Issue by martinkr** [gulpjs/gulp-cli#29]
Hey,

i just did a fresh install of gulp-cli and gulp, bothfrom branch#4 and unfortunately it's broken. The scripts worked before, now i get the following error
```
/usr/lib/node_modules/gulp-cli/lib/versioned/^4.0.0-alpha.1/index.js:34
        return gulpInst.get(task).description;
                        ^
TypeError: undefined is not a function
    at /usr/lib/node_modules/gulp-cli/lib/versioned/^4.0.0-alpha.1/index.js:34:25
    at /usr/lib/node_modules/gulp-cli/lib/shared/log/tasks.js:37:10
    at Array.forEach (native)
    at logTasks (/usr/lib/node_modules/gulp-cli/lib/shared/log/tasks.js:23:8)
    at /usr/lib/node_modules/gulp-cli/lib/versioned/^4.0.0-alpha.1/index.js:33:14
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
```
with
```
gulp-cli@0.2.0
gulp@4.0.0-alpha.1
```

Cheers!

**Comment by phated** [comment]
Duplicate of https://github.com/gulpjs/gulp/issues/1112 that was attempted to be fixed in #28 but I am on vacation and not sure I like the implementation in that PR.

**Comment by clintwood** [comment]
@phated, did you look at the updates based on your comments?
